src/main/java/org/broadinstitute/hellbender/engine/ReferenceDataSource.java

```diff
@@ -99,19 +67,9 @@ public ReferenceSequence queryAndPrefetch( final String contig, final long start
      *
      * @return SAMSequenceDictionary for this reference
      */
-    public SAMSequenceDictionary getSequenceDictionary() {
-        return reference.getSequenceDictionary();
-    }
+    public SAMSequenceDictionary getSequenceDictionary();
+
+    @Override
+    public void close();
```

Comment:
This override changes the signature to indicate that ReferenceDataSource.close may only throw RuntimeExceptions. This is already assumed in several places in the existing code so I thought it best to leave this in place.